### ISSUE TITLE
NVDA can not find the ESpeak synthesizer.

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#190]
**Reported by jason on 17 Jun 2008 00:55 UTC**
I'm using the NVDA 0.6p1 beta. ESpeak is included in Nvda, however when I try to use the synth, it says that NVDA can not find the espeak driver. If I use the temporary coppy that starts durring the instal process, espeak works.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 17 Jun 2008 02:34 UTC**
Are you using the portable or installer version of 0.6p1? I assume you are installing with the same installer in which eSpeak works correctly, but just checking.

Check the following:
 1. Navigate to your NVDA installation folder, probably ```c:\Program Files\nvda```.
 2. Open the synthDrivers folder.
 3. Check that the following files/directories exist:
  * espeak-data
  * _espeak.py
  * espeak.dll
  * espeak.py
If any of these don't exist, it would seem that your installation of NVDA failed for some reason. Try reinstalling.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 11 Aug 2008 23:01 UTC**
This works fine for me and most other users, and there has been no follow up on this bug. Closing as worksforme; please reopen if the problem persists.
